project/frontend/src/components/CompetitionCard/CompetitionCard.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from "react";
+import classes from "./CompetitionCard.module.css";
+import LinkButton from "../UI/LinkButton/LinkButton";
+
+const CompetitionCard = (props) => {
+  const getNumberWithOrdinal = (n) => {
+    var s = ["th", "st", "nd", "rd"],
+      v = n % 100;
+    return n + (s[(v - 20) % 10] || s[v] || s[0]);
```

Comment:
Might want a comment explaining this black magic

Reply:
True lmao

Reply:
Done